Fixes following QA issues:
 * QA Notice: This package installs one or more .desktop files that do not
 * pass validation.
 * 
 * 	/usr/share/applications/inputmethods/xkbd.desktop: error: (will be fatal in the future): value "xkbd.png" for key "Icon" in group "Desktop Entry" is an icon name with an extension, but there should be no extension as described in the Icon Theme Specification if the value is not an absolute path
 * 	/usr/share/applications/inputmethods/xkbd.desktop: error: value "Panel;Utility;MB" for key "Categories" in group "Desktop Entry" contains an unregistered value "MB"; values extending the format should start with "X-"
 * 	/usr/share/applications/inputmethods/xkbd.desktop: error: value "Panel;Utility;MB" for key "Categories" in group "Desktop Entry" contains an unregistered value "Panel"; values extending the format should start with "X-"
 * 
--- a/xkbd.desktop	2003-09-29 20:29:35.000000000 +0300
+++ b/xkbd.desktop	2017-02-16 16:23:40.927035269 +0200
@@ -3,6 +3,6 @@
 Comment=Virtual Keyboard
 Exec=xkbd
 Type=Application
-Icon=xkbd.png
-Categories=Panel;Utility;MB
+Icon=xkbd
+Categories=Utility;Accessibility
 X-MB-INPUT-MECHANSIM=True
